pkg/security/podsecuritypolicy/util/util.go

```diff
@@ -242,3 +242,25 @@ func EqualStringSlices(a, b []string) bool {
 	}
 	return true
 }
+
+func IsOnlyServiceAccountTokenSources(v *api.ProjectedVolumeSource) bool {
+	for _, s := range v.Sources {
```

Comment:
oh, add a `if s.Secret != nil { return false }`, with accompanying tests

Reply:
that's right.